Makefile

```diff
@@ -43,6 +43,5 @@ test-data:
 	rm -rf $(testDataDir)
 	mkdir -p $(tempDir)
 	git clone -b ${branchName} --depth 1 --single-branch ${githubRepoLink} ${gitDataDir}
-	cp ${gitDataDir}rac-experiments-v3.json ${testDataDir}
-	cp -r ${gitDataDir}assignment-v2 ${testDataDir}
```

Comment:
this was only pulling RAC-based test files. generalize to load all.

Reply:
I switched it to only loading ufc files since the rac tests are not relevant in other SDKs

Reply:
Yep but this PR is additive so I need to continue testing RAC files here.